src/main/java/org/jenkinsci/plugins/github_branch_source/Endpoint.java

```diff
@@ -59,13 +57,7 @@
     /**
      * Common prefixes that we should remove when inferring a display name.
      */
-    private static final String[] COMMON_PREFIX_HOSTNAMES = {
-            "git.",
-            "github.",
-            "vcs.",
-            "scm.",
-            "source."
-    };
+    private static final String[] COMMON_PREFIX_HOSTNAMES = { "git.", "github.", "vcs.", "scm.", "source." };
```

Comment:
Preferred the previous form, which makes it easier to review diffs to the list.

Reply:
@jglick 
Agreed, but :shrug: .
Do we want all array initializers to have one line per item?  Or is this a matter of preference for this specific case?

Reply:
Initializers which might reasonably be edited in the future.

Reply:
Addressed by forcing expansion using a line comment.  A little hacky but lets us keep most arrays unexpanded, which seems to be the more generally expected behavior.